client/viame-web-common/components/AttributesPanel.vue

```diff
@@ -283,35 +290,12 @@ export default defineComponent({
             </template>
             <span>Add a new Confidence Pair</span>
           </v-tooltip>
-          <v-tooltip
-            open-delay="200"
-            bottom
-            max-width="200"
-          >
-            <template #activator="{ on }">
-              <v-btn
-                v-if="activeSettings.confidencePairs"
-                small
-                icon
-                class="pb-2 ml-2"
-                :color="activeSettings.confidencePairs ? 'accent': ''"
-                v-on="on"
-                @click="toggleActiveSettings('confidencePairs')"
-              >
-                <v-icon
-                  small
-                >
-                  mdi-pencil
-                </v-icon>
-              </v-btn>
-            </template>
-            <span>Edit ConfidencePairs</span>
-          </v-tooltip>
```

Comment:
I removed this because it was just a button that enabled another button.

Reply:
The idea was that like the others it could toggle you into an edit mode that allowed to to edit the confidencePairs.  It may make more sense to use the single item editing on click like with the other items.